_includes/_meta_information.html

```diff
@@ -12,7 +12,7 @@
 				<time class="icon-calendar pr20" datetime="{{ page.date | date: "%Y-%m-%d" }}" itemprop="datePublished"> {{ page.date | date: "%Y-%m-%d" }}</time>
 				{% endif %}
 
-				{% if page.categories %}<span class="icon-archive pr20"> {{ page.categories | join: ' · ' | upcase }}{% endif %}</span>
+				{% if page.categories and page.categories.size > 0 %}<span class="icon-archive pr20"> {{ page.categories | join: ' · ' | upcase }}{% endif %}</span>
```

Comment:
If it's an empty array or a an array with items then `{% if page.categories.size %}` should be enough; or if you want to be really nice: `{% if page.categories != empty %}`. I'm using the latter https://github.com/TWiStErRob/twisterrob.github.io/blob/master/_includes/meta_information.html#L15